Dockerfile

```diff
@@ -16,6 +16,15 @@ RUN go build -o /grpcurl \
     -ldflags "-w -extldflags \"-static\" -X \"main.version=$(cat VERSION)\"" \
     ./cmd/grpcurl
 
+FROM alpine:3 as alpine
```

Comment:
what is alpine:3 in particular?

Reply:
@dragonsinth  This will use the latest stable version of alpine without jumping up major versions.